src/components/modals/FormStepSaveModal.js

```diff
@@ -21,6 +23,17 @@ const initialState = {
   isSaving: false,
 };
 
+const emailValidationSchema = z.object({
+  email: z.string().email(),
+});
+
+const FIELD_LABELS = defineMessages({
```

Comment:
typically you do a construct like this so that the field label and the label mentioned in the error message are the same, so you should also update https://github.com/open-formulieren/open-forms-sdk/pull/692/files#diff-505e57d8cf61bc5d12e7aca0402abc93e96f76e868e894629d8cb3b5fe7c4f7cR186 to read:

```jsx
            <EmailField
              name="email"
              isRequired
              label={intl.formatMessage(FIELD_LABELS.email)}
              description={
                <FormattedMessage
                  description="Form save modal email field help text"
                  defaultMessage="The email address where you will receive the resume link."
                />
              }
            />
```